homeassistant/helpers/entity_platform.py

```diff
@@ -201,14 +202,28 @@ async def async_setup(
         if not hasattr(platform, "async_setup_platform") and not hasattr(
             platform, "setup_platform"
         ):
-            self.logger.error(
+            self.logger.error(  # Logging this is deprecated (replaced by issue) and can be removed in 2023.8.0
```

Comment:
Maybe we can remove this logging right away?

Reply:
We often have both.